hypervisor/arch/x86/Kconfig

```diff
@@ -54,11 +56,20 @@ config LOG_BUF_SIZE
 
 config LOG_DESTINATION
 	int "Bitmap of consoles where logs are printed"
+	range 0 7
 	default 7
+	help
+	A bitmap indicating the destinations of log messages. Currently there
+	are 3 destinations available. Bit 0 represents the serial console, bit
+	1 the SOS ACRN log and bit 2 NPK log.
 
 config CPU_UP_TIMEOUT
 	int "Timeout in ms when bringing up secondary CPUs"
```

Comment:
Thanks for pointing out the inconsistency. A double check on the code indicates that CPU_UP_TIMEOUT is still in ms (''1000 * CPU_UP_TIMEOUT'' is in us). Since this has been merged already, I'll create another patch adding more help info as long as fixing this and the other comments below.